Update README.MD for prototype version 7 documentation

bettymw opened the pull request:
# PR Checklist

- [ ] If you are proposing a new decision record document, used the right template for that
      - ([ADR](https://github.com/alphagov/forms/blob/main/ADR/ADRXXX-architecture-decision-record-template.md), [decision-record](https://github.com/alphagov/forms/blob/main/decision-record/DRXXX-decision-record-template.md), engagement, [research](https://github.com/alphagov/forms/blob/main/research/YYYY-MM-DD-template.md))
- [ ] Set yourself as the Assignee
- [ ] Tag anyone you would like to review, or @forms-design or @forms-devs
- [ ] Fill in the template below


---

# What

Describe what you have changed and why.

# How to review

Describe the steps required to test the changes.

For example:

1. Semantic: Do you agree with the changes?
1. Syntactic: Spelling, grammar, etc.

# Who can review

Describe who can review the changes. Or more importantly, list the people
that can't review, because they worked on it.

bettymw commented:
I have documented the pages for the forms admin and preview forms runner for the prototype version 7. 

At first it appeared as if this version was what was tested in sprint 9, but I think this version was not tested. What was tested in sprint 9 has some slight variations on some pages. I am still looking at the murals and other documentation to ascertain and then I will update on this version.

christophercameron-ixd commented:
Sorry, accidentally closed this. 

Thanks for doing this Betty. I think the screenshots could do with more detail about what is on the screens as well as some of the details of what was changed based on the [change logs](https://github.com/alphagov/forms-prototypes/releases/tag/v0.0.7).

I think you are right this wasn’t tested. It was the content and changes based on Sprint 9 (prototype version 6) which still needs written up properly.

bettymw commented:
> I think the screenshots could do with more detail about what is on the screens as well as some of the details of what was changed based on the [change logs](https://github.com/alphagov/forms-prototypes/releases/tag/v0.0.7).

You mean like mention what was changed according to the change logs, per screen in comparison to the previous version 6? Or elaborate the changes in a separate section?

christophercameron-ixd commented:
Yeah pretty much just that. 
One way I did this previously was to write up a quick comparison of the screens like https://github.com/alphagov/forms/tree/main/design/prototype-version-4#what-we-changed-from-the-previous-version and a quick summary of those things that changed https://github.com/alphagov/forms/tree/main/design/prototype-version-4#some-things-we-changed-since-last-time 

It might also be worth noting to exclude email addresses and names from the write up. So anywhere my email pops up for instance you could just replace it with email@example.gov.uk or something similar if that’s ok :)

chao-xian commented:
@bettymw @christophercameron-ixd Hello this has been open for a while can it be progressed any further?

bettymw commented:
@chao-xian will be progressing this documentation this week

bettymw commented:
Hi @christophercameron-ixd pls have a review of the updates I have made:
- have added more explanations of the changes on the narration below the screenshots for different pages.
- Added a new section for new journeys we introduced  
- created a file to show the changes on the task list page and linked to the main version 7 file 
- updated the context section
- fixed typos and grammatical errors

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+
+### Add and edit questions (on a live form) 
+
+![add and edit questions to a live form page. Screenshot](screenshots/802-form-index-live.png)
```
```suggestion
![Add and edit questions of a live form page. Screenshot](screenshots/802-form-index-live.png)
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+### Add and edit questions (on a live form) 
+
+![add and edit questions to a live form page. Screenshot](screenshots/802-form-index-live.png)
+*Page with a blue notification banner with title “Important“, followed by “Apply for a juggling licence” caption above the heading that says “Add and edit your questions”.*
```
```suggestion
*Page with a blue notification banner titled “Important”, followed by “Apply for a juggling licence” caption above the heading “Add and edit your questions”.*
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+![add and edit questions to a live form page. Screenshot](screenshots/802-form-index-live.png)
+*Page with a blue notification banner with title “Important“, followed by “Apply for a juggling licence” caption above the heading that says “Add and edit your questions”.*
+
+At the start of the page is a blue notification banner with title “Important“. Inside the notification banner there is a secondary heading saying “Any changes you make to a live form will be updated in the form immediately.“ 
```
```suggestion
Inside the notification banner there is a secondary heading saying “Any changes you make to a live form will be updated in the form immediately.“ 
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+
+This is followed by a paragraph describing how any changes will affect people filling the form “This could have an impact on people who are filling in the form at the same time. They may lose any answers they have already provided and may need to start again.“
+
+Below the heading is a green status tag “LIVE“ to let the form creator know this is a live form. 
```
```suggestion
Below the heading is a green “LIVE“ status tag letting the form creator know this is a ‘live’ form. 
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+There is a green “Add a question” button followed by a secondary heading, “Your questions" and below is a link "Preview the form in a new tab". 
+
+There is a list of questions created and saved for this form. 
+The first question is "What is your name?". To the right is a grey “Move down” button and below it a link "edit".
```
```suggestion
The first question is “What is your name?”. To the right is a grey “Move down” button and below it an “Edit” link.   
  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+
+There is a list of questions created and saved for this form. 
+The first question is "What is your name?". To the right is a grey “Move down” button and below it a link "edit".
+There is a section break
```
Not sure this is needed?

bettymw replied:
Yes, have deleted

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+The first question is "What is your name?". To the right is a grey “Move down” button and below it a link "edit".
+There is a section break
+The second question is "What is your date of birth?". To the right is a grey “Move up” button, next to it a grey "Move down" button. Bellow the "Mode down" is a link "edit".
+There is a section break
```
Not sure this is needed?

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+There is a list of questions created and saved for this form. 
+The first question is "What is your name?". To the right is a grey “Move down” button and below it a link "edit".
+There is a section break
+The second question is "What is your date of birth?". To the right is a grey “Move up” button, next to it a grey "Move down" button. Bellow the "Mode down" is a link "edit".
```
```suggestion
The second question is “What is your date of birth?”. To the right is a grey “Move up” button, next to it a grey “Move down” button. Below the “Move down” button is an “Edit” link.   
  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+There is a section break
+The second question is "What is your date of birth?". To the right is a grey “Move up” button, next to it a grey "Move down" button. Bellow the "Mode down" is a link "edit".
+There is a section break
+The third queston is "What is your National Insurance number?". To the right is a grey "Move up" button and below is a link "edit".
```
```suggestion
The third question is “What is your National Insurance number?”. To the right is a grey “Move up” button and below is an “Edit” link.   
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+<br>
+
+## Some things we changed since last time
+- Added draft and live status tags to let a user know what state a form is in
```
```suggestion
- Added ‘draft’ and ‘live’ status tags to let form creators know what state a form is in  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+## Some things we changed since last time
+- Added draft and live status tags to let a user know what state a form is in
+- Content changes on the example heading and text input label on the edit confirmation / form submitted page
+- Content changes on the example heading and text input label on the edit add a declaration / check your answers page
```
```suggestion
- Content changes on the ‘example’ heading and textarea label on the “Add a declaration” - check your answers page
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+
+## Some things we changed since last time
+- Added draft and live status tags to let a user know what state a form is in
+- Content changes on the example heading and text input label on the edit confirmation / form submitted page
```
```suggestion
- Content changes on the ‘example’ heading and textarea label on the “Form submitted page” - confirmation of a submission page  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+- Added draft and live status tags to let a user know what state a form is in
+- Content changes on the example heading and text input label on the edit confirmation / form submitted page
+- Content changes on the example heading and text input label on the edit add a declaration / check your answers page
+- Updated the second level heading 'Add and edit your questions' page to 'Your questions'
```
```suggestion
- Updated the second level heading “Add and edit your questions” page to “Your questions”  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+- Content changes on the example heading and text input label on the edit confirmation / form submitted page
+- Content changes on the example heading and text input label on the edit add a declaration / check your answers page
+- Updated the second level heading 'Add and edit your questions' page to 'Your questions'
+- Content updates on the 'edit question page' 
```
```suggestion
- Content updates on the “Edit question” page  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+- Content changes on the example heading and text input label on the edit add a declaration / check your answers page
+- Updated the second level heading 'Add and edit your questions' page to 'Your questions'
+- Content updates on the 'edit question page' 
+- [Made changes to the Create a form (task list) page](https://github.com/alphagov/forms/tree/documenting-prototype-version-7/design/prototype-version-7/task-list)
```
```suggestion
- [Made changes to the “Create a form” task list page](task-list)  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+### Preview question 1
+
+![Preview What is your name question page. Screenshot](screenshots/1001-page-preview-new-tab-1.png)
+*Page with “What is your name?” question as a label for a text input. There is a green “Continue” button and a details summary text “Get help with this form“ at the bottom.*
```
```suggestion
*Page with “What is your name?” question as the label for a text input. There is a green “Continue” button and a details component with the text “Get help with this form“ at the bottom.*
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+### Preview final question
+
+![Preview What is your National Insurance number question page. Screenshot](screenshots/1002-page-preview-new-tab-3.png)
+*Page with “What is your National Insurance number?” question as a label for a text input, followed below by a green “Continue” button.*
```
```suggestion
*Page with “What is your National Insurance number?” question as the label for a text input, followed below by a green “Continue” button.*
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+![Preview What is your National Insurance number question page. Screenshot](screenshots/1002-page-preview-new-tab-3.png)
+*Page with “What is your National Insurance number?” question as a label for a text input, followed below by a green “Continue” button.*
+
+There is a details summary link text that says “Get help with this form“ that is expanded to reveal the third level heading “Email“ and an example of a contact email address “support@email.gov.uk“. We added this details component of the contact details that can be used by a person filling a form to get help.
```
```suggestion
There is a details component that says “Get help with this form“ that is now expanded. This reveals a third level heading, “Email“, and an example of a contact email address “support@email.gov.uk“. We added this details component for the contact details that can be used by a person filling a form to get help.  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+
+There is a details summary link text that says “Get help with this form“ that is expanded to reveal the third level heading “Email“ and an example of a contact email address “support@email.gov.uk“. We added this details component of the contact details that can be used by a person filling a form to get help.
+
+This is the contact details added by the form creator within the admin side of the builder, meaning it can be customised. The form creator can also provide a link to contact support form or a phone number or all three options.
```
```suggestion
This is the contact details added by the form creator within the admin side of the builder. The form creator can choose to provide a link to a contact support form or a phone number or any combination of the three contact options.  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+![Preview check your answers page. Screenshot](screenshots/1003-check-answers-page-preview-new-tab.png)
+*Page with “Check your answers before submitting your form” heading followed by a summary list component.*
+
+The summary list component lists rows showing the form’s full questions. There is a space to the right where the form filler's answer would appear: this has placeholder text saying “Not completed”. Finally, there is a “Change” link for the form filler to correct or change any answer they feel is incorrect.
```
```suggestion
Each row shows the full question text added to the form followed by a space to the right where the form filler’s answer would appear. This currently has placeholder text saying “Not completed”. Finally, there is a “Change” link for the form filler to change any answer they feel is incorrect.  
```

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,505 @@
+___
+
+[Back to the top](#prototype-version-7)
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```
```suggestion
```

christophercameron-ixd reviewed (changes requested):
A fair few tweaks or corrections to look at, but otherwise it looks good. 
Thanks again for doing this!

bettymw commented on design/prototype-version-7/task-list/README.md:
```diff
@@ -0,0 +1,125 @@
+The second row has a link to “Add and edit your questions“ and a grey highlighted tag on the far right of the row that reads “not started“.
+
+The third row has a link to “Review summary page and add declaration“ and a grey highlighted tag on the far right of the row that reads “not started“.
+
```
yes I accidentally missed this, its now added

bettymw reviewed (commented):
I have made changes and accepted the review suggestions

bettymw reviewed (commented):
Made the necessary changes

christophercameron-ixd commented on design/prototype-version-7/README.MD:
```diff
@@ -1 +1,480 @@
+
+## Context
+
+> In this prototype version, we introduced new user journeys and made content improvements to some pages, based on findings from usability testing in Sprint 9 on prototye version 6. 
```
```suggestion
> In this prototype version, we introduced new user journeys and made content improvements to some pages, based on findings from usability testing in Sprint 9 on prototype version 6. 
```

christophercameron-ixd commented on design/prototype-version-7/task-list/README.md:
```diff
@@ -0,0 +1,140 @@
+
+## What we tested last time
+
+![“Create a form” task list page. Screenshot](../screenshots/003-Create-a-form-Apply-for-a-juggling-licence.png)
```
```suggestion
![“Create a form” task list page. Screenshot](../../prototype-version-6/screenshots/003-Create-a-form-Apply-for-a-juggling-licence.png)
```
I think this will link to V6 screenshot - if that’s correct

bettymw replied:
yes it's meant to show the task list from V6 (which was tested)

christophercameron-ixd commented on design/prototype-version-7/task-list/README.md:
```diff
@@ -0,0 +1,140 @@
+
+<br>
+
+![Step “1. Create your form” tasks. Screenshot](../screenshots/003-1-Task-list-page-tags-focus-create-your-form.png)
```
```suggestion
![Step “1. Create your form” tasks. Screenshot](../../prototype-version-6/screenshots/003-1-Task-list-page-tags-focus-create-your-form.png)
```
I will make sure to add this screenshot into V6 or I will update the link when writing up V6

christophercameron-ixd commented on design/prototype-version-7/task-list/README.md:
```diff
@@ -0,0 +1,140 @@
+<br>  
+
+- Changed step title from “4. Publish your form“ to “4. Make your form live“.
+- Removed the final task ”How to publish the form on GOV.UK“ because the guidance to publish a form on GOV.UK was provided in the [“Make your form live“](design/prototype-version-7/screenshots/701-make-your-form-live.png) page.
```
Is this supposed to link to a screenshot elsewhere on the page, or should it open the image file? 
This would open the image file: 
```suggestion
- Removed the final task ”How to publish the form on GOV.UK“ because the guidance to publish a form on GOV.UK was provided in the [“Make your form live“](../screenshots/701-make-your-form-live.png) page.
```

bettymw replied:
It should open the make your form live screenshot, which can be found on the prototype version 7 read me

christophercameron-ixd commented on design/prototype-version-7/task-list/README.md:
```diff
@@ -0,0 +1,140 @@
+
+<br> 
+
+![task list page for a live form. Screenshot](https://github.com/alphagov/forms/blob/documenting-prototype-version-7/design/prototype-version-7/screenshots/703-create-form-live.png)
```
```suggestion
![task list page for a live form. Screenshot](../screenshots/703-create-form-live.png)
```
